imap_processing/idex/l1/idex_l1.py

```diff
@@ -45,420 +41,75 @@ class Scitype(IntEnum):
+        TriggerDescription("tof_high_trigger_level", "IDX__TXHDRHGTRIGLVL"),
+        TriggerDescription("tof_high_trigger_num_max_1_2", "IDX__TXHDRHGTRIGNMAX12"),
+        TriggerDescription("tof_high_trigger_num_min_1_2", "IDX__TXHDRHGTRIGNMIN12"),
+        TriggerDescription("tof_high_trigger_num_min_1", "IDX__TXHDRHGTRIGNMIN1"),
+        TriggerDescription("tof_high_trigger_num_max_1", "IDX__TXHDRHGTRIGNMAX1"),
+        TriggerDescription("tof_high_trigger_num_min_2", "IDX__TXHDRHGTRIGNMIN2"),
+        TriggerDescription("tof_high_trigger_num_max_2", "IDX__TXHDRHGTRIGNMAX2"),
+        TriggerDescription("tof_low_trigger_level", "IDX__TXHDRLGTRIGLVL"),
+        TriggerDescription("tof_low_trigger_num_max_1_2", "IDX__TXHDRLGTRIGNMAX12"),
+        TriggerDescription("tof_low_trigger_num_min_1_2", "IDX__TXHDRLGTRIGNMIN12"),
+        TriggerDescription("tof_low_trigger_num_min_1", "IDX__TXHDRLGTRIGNMIN1"),
+        TriggerDescription("tof_low_trigger_num_max_1", "IDX__TXHDRLGTRIGNMAX1"),
+        TriggerDescription("tof_low_trigger_num_min_2", "IDX__TXHDRLGTRIGNMIN2"),
+        TriggerDescription("tof_low_trigger_num_max_2", "IDX__TXHDRLGTRIGNMAX2"),
+        TriggerDescription("tof_mid_trigger_level", "IDX__TXHDRMGTRIGLVL"),
+        TriggerDescription("tof_mid_trigger_num_max_1_2", "IDX__TXHDRMGTRIGNMAX12"),
+        TriggerDescription("tof_mid_trigger_num_min_1_2", "IDX__TXHDRMGTRIGNMIN12"),
+        TriggerDescription("tof_mid_trigger_num_min_1", "IDX__TXHDRMGTRIGNMIN1"),
+        TriggerDescription("tof_mid_trigger_num_max_1", "IDX__TXHDRMGTRIGNMAX1"),
+        TriggerDescription("tof_mid_trigger_num_min_2", "IDX__TXHDRMGTRIGNMIN2"),
+        TriggerDescription("tof_mid_trigger_num_max_2", "IDX__TXHDRMGTRIGNMAX2"),
+        TriggerDescription(
+            "low_sample_coincidence_mode_blocks", "IDX__TXHDRLSTRIGCMBLOCKS"
+        ),
+        TriggerDescription("low_sample_trigger_polarity", "IDX__TXHDRLSTRIGPOL"),
+        TriggerDescription("low_sample_trigger_level", "IDX__TXHDRLSTRIGLVL"),
+        TriggerDescription("low_sample_trigger_num_min", "IDX__TXHDRLSTRIGNMIN"),
+        TriggerDescription("low_sample_trigger_mode", "IDX__TXHDRLSTRIGMODE"),
+        TriggerDescription("tof_low_trigger_mode", "IDX__TXHDRLSTRIGMODE"),
+        TriggerDescription("tof_mid_trigger_mode", "IDX__TXHDRMGTRIGMODE"),
+        TriggerDescription("tof_high_trigger_mode", "IDX__TXHDRHGTRIGMODE"),
+        TriggerDescription("detector_voltage", "IDX__TXHDRHVPSHKCH0"),
+        TriggerDescription("sensor_voltage", "IDX__TXHDRHVPSHKCH1"),
+        TriggerDescription("target_voltage", "IDX__TXHDRHVPSHKCH2"),
+        TriggerDescription("reflectron_voltage", "IDX__TXHDRHVPSHKCH3"),
+        TriggerDescription("rejection_voltage", "IDX__TXHDRHVPSHKCH4"),
+        TriggerDescription("detector_current", "IDX__TXHDRHVPSHKCH5"),
     ]
 }
 
 
+def create_idex_attr_obj(data_version: str) -> ImapCdfAttributes:
```

Comment:
Personal preference and nitpicky here, but I would prefer this be named `get_idex_attrs` -- the way it is currently named reads to me like some sort of class is being created that will later be modified. Might just be a me problem though 😂, it's totally up to you.